changelog.md

```diff
@@ -15,6 +15,7 @@ SciReactUI Changelog
 - Breadcrumbs component takes optional linkComponent prop for page routing. 
 - Navbar, NavLink and FooterLink will use routing library for links if provided with linkComponent and to props.
 - Navbar uses slots for positioning elements. Breaking change: elements must now use rightSlot for positioning to the far right.
+- Footer uses slots for positioning elements. Breaking change: elements must now use centralSlot and rightSlot for positioning to the centre and far right.
```

Comment:
You can remove the centreSlot part here as it's not a breaking change. keep the rightSlot part.